### ISSUE TITLE
3.1.2 sentry-expo not working in web

### ISSUE THREAD

**Issue by SteveAquino** [expo/sentry-expo#179]
### Summary

I updated to the latest version of sentry-expo (3.1.2) and now the application will not run or build for web.  It looks like it's failing to parse some syntax in the source files.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Web

### SDK Version (managed workflow only)

40

### Environment

```
  Expo CLI 4.2.1 environment info:
    System:
      OS: macOS 11.2.1
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 14.15.2 - /usr/local/bin/node
      npm: 6.14.9 - /usr/local/bin/npm
    Managers:
      CocoaPods: 1.10.0 - /Users/stevenaquino/.rvm/gems/ruby-2.7.2/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    IDEs:
      Xcode: 12.4/12D4e - /usr/bin/xcodebuild
    npmPackages:
      expo: ^40.0.0 => 40.0.0
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-40.0.0.tar.gz => 0.63.2
      react-native-web: ~0.13.12 => 0.13.18
    npmGlobalPackages:
      expo-cli: 4.0.15
    Expo Workflow: managed
```

### Reproducible demo or steps to reproduce from a blank project

1. Update sentry-expo to 3.1.2 in package.json
2. run `npm install`
3. run `npm start`
4. press "w" to open the web browser.

![Screen Shot 2021-03-05 at 4 33 13 PM](https://user-images.githubusercontent.com/539839/110175921-80a68b80-7dd0-11eb-827f-d4649d530c37.png)

**Comment by cruzach** [comment]
fixed in https://github.com/expo/sentry-expo/commit/f1f63f2156e6d9aebcd416dbfcb684de6f3bfc90

published in v3.1.3

**Comment by SteveAquino** [comment]
@cruzach Wow, that was really fast and unexpected.  Thanks so much for the quick response!  I just tested this version and it seems to be working now.
